tests/mocks/mock_multi_text_input_task.py

```diff
@@ -1,4 +1,4 @@
-from attrs import define
+from attrs import Attribute, define
```

Comment:
Why are you adding an import `Attribute` here even though no use is added in the rest of the file?

Reply:
Accidental import, removed.